python/tskit/util.py

```diff
@@ -60,7 +60,10 @@ def is_unknown_time(time):
     """
     As the default unknown mutation time (:const:`UNKNOWN_TIME`) is a specific NAN value,
     equality always fails. This method compares the bitfield such that unknown times can
-    be detected. Either single floats can be passed or lists/arrays.
+    be detected. Either single floats can be passed or lists/arrays. Note that many types
```

Comment:
It's a bit more accurate to say that there are many different NaN *values*, but only one specific NaN matches UNKNOWN_TIME. Maybe link to the NaN wikipedia page so people can read about it: https://en.wikipedia.org/wiki/NaN